src/Workload/Microsoft.Maui.Controls.Sdk/Sdk/BundledVersions.in.targets

```diff
@@ -55,13 +56,19 @@
 
   <!-- These implicit <PackageReference/> pull dependencies from NuGet transitively -->
   <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
-    <PackageReference Include="Microsoft.Maui.Extensions" Version="$(MicrosoftMauiSdkVersion)" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.Maui.Extensions" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
   <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' ">
-    <PackageReference Include="Microsoft.Maui.Dependencies" Version="$(MicrosoftMauiSdkVersion)" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.Maui.Dependencies" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' ">
+    <PackageReference Include="Microsoft.Maui.Controls.Build.Tasks" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true' ">
+    <PackageReference Include="Microsoft.Maui.Resizetizer.Sdk" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
```

Comment:
This is the major change here -- these two packs are `library-packs` and an implicit `@(PackageReference)` is brought into Maui projects.

Reply:
@jonathanpeppers can `$(MauiVersion)` be a floating version? is there a way this could be discoverable from NuGet package manager?

Reply:
I don't think there is a way for the NuGet package manager to show this... I guess we could make some package that you add to your project that could do it? Some Microsoft.Maui package could set `MauiVersion` in its MSBuild targets inside.

I asked about floating versions here: https://github.com/dotnet/docs/pull/25813#discussion_r697660711

I could try and just see if it works.

Reply:
Floating numbers could be a reasonable answer.  I do like the idea of some other meta package that we publish too that just sets the version, that's a simple enough work around that would surface in the package manager updates UI.